test/pass/rocm-list-split.cmake

```diff
@@ -1,4 +1,4 @@
-
+cmake_policy(SET CMP0011 NEW)
```

Comment:
What is this for? Maybe this should be moved to the test.cmake file instead if its useful for all the tests.

Reply:
This was to reduce the noise when running "make pass_rocm_list_plit" to run a single test.